app/assets/src/api/bulk_downloads.js

```diff
@@ -17,7 +17,7 @@ export const getBulkDownload = bulkDownloadId =>
 export const getPresignedOutputUrl = bulkDownloadId =>
   get(`/bulk_downloads/${bulkDownloadId}/presigned_output_url`);
 
-export const validateSampleIdsForBulkDownload = sampleIds =>
+export const validateSampleIds = sampleIds =>
   postWithCSRF("/bulk_downloads/validate_sample_ids", {
     sampleIds: sampleIds,
   });
```

Comment:
I'm glad you're also finding a use for this function :) We should probably refactor the backend part to be more generalized, like changing the route and function to live on the samples controller. But I think that can wait for an API review.

Reply:
This is a good point! I am actually in favor of doing this small low risk refactors as we find them, before they grow to be used in multiple places.